packages/app/src/cli/services/app-logs/poll-app-logs.ts

```diff
@@ -68,9 +66,9 @@ export const pollAppLogs = async ({
           }).catch((error) => {
             outputDebug(`Unexpected error during polling: ${error}}\n`)
           })
-        }, POLLING_BACKOFF_INTERVAL_MS)
+        }, POLLING_THROTTLE_RETRY_INTERVAL_MS)
       } else {
-        throw new Error(`Error while fetching: ${responseText}`)
+        throw new Error(`Unhandled bad response: ${response.status}`)
```

Comment:
I'm a little confused here- in the PR description it says that we re-enqueue no matter what, but it seems like we're not re-queuing 500s based on this if statement